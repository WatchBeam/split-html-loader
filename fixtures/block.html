<!-- start platform: xbox -->
<one-fish></one-fish>
<two-fish></two-fish>
<!-- end platform: xbox -->
<!-- start platform: desktop -->
<red-fish></red-fish>
<blue-fish></blue-fish>
<!-- end platform: desktop -->
<not-a-fish></not-a-fish>
