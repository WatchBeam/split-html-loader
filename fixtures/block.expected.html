<!-- start platform: xbox -->
<one-fish></one-fish>
<two-fish></two-fish>
<!-- end platform: xbox -->
<!-- start platform: desktop --><!-- 6 nodes snipped by split-html --><!-- end platform: desktop -->
<not-a-fish></not-a-fish>
